tensilelite/Tensile/Tests/common/gemm/gfx950/dtl.yaml

```diff
@@ -84,12 +84,12 @@ BenchmarkProblems:
         - SourceSwap: [1]#[0, 1]
         - LocalReadVectorWidth: [8]
         - DirectToLds: [1]
-        - AssertSummationElementMultiple: [4]
+        - AssertSummationElementMultiple: [1, 4]
```

Comment:
Do we have any test case with GSU?
I would like to know if this logic really works with GSU.
Let's say, if ASEM=4, K%DepthU=4, GSU=2, in this case, the actual K size assigned to each CU is K/GSU.
Then, K/GSU is not a multiple of 4.

I am wondering if ASEM really works as expected with GSU.

Reply:
I quickly checked some cases with `GSU > 1` and it looks like there are some predicates that enforces K so that `K / GSU` satisfies the ASEM condition. :thinking:

Reply:
OK. Thanks for confirmation.
Interesting.
Please add some GSU test cases.